### PR TITLE
[ML] Split Live Tests Into Separate Subscriptions

### PR DESCRIPTION
**Pull request by needuv**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by scbedd — APPROVED** [review]
Love it.

**Review by pritamDas9191 — APPROVED** [review]

**Review by TonyJ1 — APPROVED** [review]

File: sdk/ml/automl.tests.yml [hunk 1]
```diff
@@ -3,4 +3,8 @@ trigger: none
 extends:
   template: ./tests.yml
   parameters:
-    TestMarkArgument: "automl_test"
\ No newline at end of file
+    TestMarkArgument: "automl_test"
+    CloudConfig:
+      Public:
```
Comment (benbp) [inline sdk/ml/automl.tests.yml:8]: This and the one below would be more consistently named as `Canary` given they target euap regions.

**Review by wangchao1230 — APPROVED** [review]

File: sdk/ml/automl.tests.yml (hunk 1 above)
Comment (needuv) [reply sdk/ml/automl.tests.yml:8]: This is likely to changed very soon to be a prod region, so I think I'll just leave it as is

**Pull request merged**